duckduckgo/macos-browser
# Update the home page to preserve the scheme of the original URL.

Task/Issue URL: https://app.asana.com/0/1199230911884351/1202894007986850/f
Tech Design URL:
CC:

**Description**:

This PR updates the new tab page to remember the scheme that was used to open URLs from history.



**Steps to test this PR**:
1. Trigger the Fire button to clear your history out
1. Visit the HTTP variant of the privacy test pages: http://privacy-test-pages.glitch.me/, click one of the links in the list to open another test page
1. Open the new tab page and click the privacy test pages link at the top of the recently visited list, verify that it opens in HTTP (**not** HTTPS). Check that the path that you visited also opens in HTTP.
1. Trigger the Fire button again
1. Visit the HTTPS privacy test pages: https://privacy-test-pages.glitch.me/, click on one of the links again
1. Open the new tab page once more and check that the privacy test pages links open with HTTPS

**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
